service/filesservice.php

```diff
@@ -143,13 +142,9 @@ protected function getFolderData($node) {
 	protected function getAllowedSubFolder($node, $nodeType) {
 		if ($nodeType === 'dir') {
 			/** @var Folder $node */
-			try {
- 				if (!$node->nodeExists($this->ignoreAlbum)) {
- 					return [$node];
- 				}
-			} catch (StorageNotAvailableException $e) {
- 				return [];
- 			}
+			if (!$node->nodeExists($this->ignoreAlbum)) {
```

Comment:
why this change? please explain - THX

Reply:
Sorry, but i diddn't change this. Anyway i updated taking files their gallery master. Same latest commit.